queries/glimmer/indents.scm

```diff
@@ -0,0 +1,22 @@
+[
+  (element_node (element_node_start))
+  (element_node_void)
+  (block_statement (block_statement_start))
+  (mustache_statement)
+] @indent.begin
+
+(element_node (element_node_end  [">"] @indent.end))
+(element_node_void "/>" @indent.end)
+[
+ ">"
+ "/>"
+ "</"
+ "{{/"
+ "}}"
+] @indent.branch
+
+(mustache_statement
+  (helper_invocation helper: (identifier) @_identifier (#lua-match? @_identifier "else"))
+  ) @indent.branch
+(mustache_statement ((identifier) @_identifier (#lua-match? @_identifier "else"))) @indent.branch
```

Comment:
Not familiar with the language, why does it have to be `lua-match?` and not `eq?` ?

Reply:
Yeah I guess it doesn't in this case. Thanks, I'll make the update here.